dashboard/app/helpers/survey_results_helper.rb

```diff
@@ -1,24 +1,16 @@
 module SurveyResultsHelper
   def show_survey?
-    # rubocop:disable Lint/UnreachableCode
-
-    # Disabled for all users on 2016-03-30 after being shown for thirty days.
-    return false
-
     # Reasons we would not show the survey.
     unless current_user
       return false
     end
-    unless current_user.teacher?
```

Comment:
Do you mean how do we revive this when we switch to the diversity survey?  Or are we going to be running multiple surveys at one time?